DataScience/ExperimentationAzure.py

```diff
@@ -157,6 +157,8 @@ def check_system():
                 other_args.append(model_fp)
             other_args.append('--min_num_features')
             other_args.append('1')
+            other_args.append('--ml_args')
```

Comment:
probably out of scope:
--ml_args is a required parameter/argument, but based on the help comment it is only used for getting feature importance. This makes me think that our cmd parsing is off or that we should be using ml_args elsewhere, but I couldn't find where that happened.

I think one or more of these needs to happen:
1) We fix the ml_args parameter to only be required if we are getting feature importance.
2) Update the help comment to indicate that ml_args is **not only** used for feature importance.
3) Check to see if ml_args is needed or used elsewhere and update help comment or code accordingly.

Nice catch!

Reply:
That's great observation.
It is not used elsewhere. Only for feature importance for now.
I would leave fixing the arguments to another PR.